nodes/config/tts.html

```diff
@@ -48,7 +13,7 @@
     },
     oneditprepare: function() {
       var node = this;
-      languages.forEach(function(language) {
+      languages().forEach(function(language) {
```

Comment:
I see no computation on the array which requires it to be returned from a function. I suggest we remove the function as well as function calls here. It should simply suffice as a variable.

`languages.forEach(...)` instead of `languages().forEach(...)`.

Reply:
I've changed to function to have this long array at the end of the file.